raiden/tests/integration/network/test_pathfinding.py

```diff
@@ -0,0 +1,89 @@
+from unittest.mock import patch
+
+import pytest
+from eth_utils import encode_hex
+
+from raiden.exceptions import ServiceRequestFailed
+from raiden.network.pathfinding import get_pfs_iou, make_iou, update_iou
+from raiden.utils import privatekey_to_address
+from raiden.utils.typing import Address, TokenNetworkAddress
+from raiden_contracts.utils.proofs import sign_one_to_n_iou
+
+
+@pytest.mark.skip(reason="TODO: vulnerable to evil PFS")
```

Comment:
I don't know how to fix this vulnerability, since our current plan is to not store the state of iou in the client and rely on the pfs for it. But you have a point here, we may need to change that.